tsconfig.json

```diff
@@ -39,6 +39,18 @@
     "src/analysis/retail/shaman/shared",
     "src/analysis/retail/warrior/arms"
   ],
+  "files": [
+    "src/analysis/retail/deathknight/unholy/CONFIG.tsx",
+    "src/analysis/retail/druid/guardian/CONFIG.tsx",
+    "src/analysis/retail/hunter/beastmastery/CONFIG.tsx",
+    "src/analysis/retail/hunter/marksmanship/CONFIG.tsx",
+    "src/analysis/retail/hunter/survival/CONFIG.tsx",
+    "src/analysis/retail/monk/windwalker/CONFIG.tsx",
+    "src/analysis/retail/paladin/protection/CONFIG.tsx",
+    "src/analysis/retail/paladin/retribution/CONFIG.tsx",
+    "src/analysis/retail/priest/shadow/CONFIG.tsx",
+    "src/analysis/retail/warrior/arms/CONFIG.tsx"
+  ],
```

Comment:
since these are explicitly imported now, this is not necessary---typescript will check them despite the exclusion.

Reply:
actually maybe i'm wrong on this :thinking: gonna leave it alone for now

Reply:
Even if they're explicitly imported, because they're in `exclude`, TypeScript will ignore them. The only way to have TypeScript typecheck the Config file is to leave them in `files`. It was a wild afternoon finding that out.

Reply:
That's really strange because the whole reason I had to comment out the imports is because of typescript typechecking everything that was imported, even if it was also excluded

Reply:
TS does appear to hang if I remove them from files so I'm not touching it though lol